src/lib.rs

```diff
@@ -155,7 +193,7 @@ impl CorsHandlerWhitelist {
         }
 
         // If we don't have an Access-Control-Request-Method header, treat as a possible OPTION CORS call
-        return self.process_possible_cors_request(req, origin)
+        return self.process_possible_cors_request(req, origin);
```

Comment:
If you're at it, can you change this to `self.process_possible_cors_request(req, origin)` (without the `return`)?